src/test/java/ru/ayaz/CommentTest.java

```diff
@@ -0,0 +1,49 @@
+package ru.ayaz;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+
+public class CommentTest {
+
+    @Test
+    public void shouldBeEqualWhenReferencesAreSame() {
```

Comment:
&#x1F34E; **Issue**
I would say that this test is redundant. The test "shouldBeEqualWhenAuthorAndTextAreSame" should be enough. If "shouldBeEqualWhenAuthorAndTextAreSame" passes then it (almost) inevitably follows that this test will pass (cause you use compare object's fields).